### ISSUE TITLE
Chapter 4

### ISSUE THREAD

**Issue by bitcoin333** [bitcoinbook/bitcoinbook#39]
Hi, Andreeas,

"and check that it is less than n-1"
correct:
"and check that it is less than n"

I think that instead of bitcoind now bitcoin-cli should be used 

Elliptic addition of two points would be also useful to see, and augment explanation that after "computing" G, 2G, 4G, 8G, 16G, etc the appropriate terms can be added.

public key for k = 1E99423A4ED27608A15A2616A2B0E9E52CED330AC530EDCC32C8FFC6A526AEDD is incorrect, bitaddress.org says 04F028892BAD7ED57D2FB57BF33081D5CFCF6F9ED3D3D7F159C2E2FFF579DC341A07CF33DA18BD734C600B96A72BBC4749D5141C90EC8AC328AE52DDFE2E505BDB

"Each public key requires 513 bytes"
correct:
"Each public key requires 65 bytes"

**Comment by enderminh** [comment]
Hi bitcoin333. Thanks for reporting this!

I have fixed the public key in the text. You are correct and the correct coordinates are:
x = F028892BAD7ED57D2FB57BF33081D5CFCF6F9ED3D3D7F159C2E2FFF579DC341A
y = 07CF33DA18BD734C600B96A72BBC4749D5141C90EC8AC328AE52DDFE2E505BDB
with y being odd.

I've also changed the uncompressed public key to require 520 bits to be consistent with how Andreas describes the compressed key to require 264 bits.

As for the first three points about n - 1,  bitcoind vs bitcoin-cli and the Elliptic addition graph, I will leave this up to Andreas to decide/fix/close, so I assigning this bug to him.
